packages/widget-v2/src/utils/clientType.ts

```diff
@@ -338,11 +338,11 @@ export enum TransferType {
 }
 
 export type SimpleStatus =
+  | ""
+  | "signing"
```

Comment:
"broadcasted" was the old equivalent of "" since we weren't actually ever rendering anything for "broadcasted"

Reply:
there's an argument to be made that we should change "" to something else like "unconfirmed"

Reply:
actually i like that, ill update my code